banwebScrape.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/python3
+#!/usr/bin/env python
```

Comment:
Should still be ` #!/usr/bin/env python3` if we're switching to environments.  We can't / shouldn't force a virtual environment, that's mostly a development thing.

Also if it's being used from a system with python 2 as default *cough RHEL* it should still work.